src/lib.rs

```diff
@@ -35,14 +35,27 @@
 //! ```
 
 extern crate libc;
+
 #[cfg(target_os = "redox")]
 extern crate syscall;
 
-#[cfg(windows)] extern crate winapi;
+#[cfg(windows)]
+extern crate winapi;
+
+
 #[cfg(any(unix, target_os = "redox"))] use std::os::unix::prelude::*;
-#[cfg(unix)] use libc::{c_char, c_int, timeval};
 #[cfg(windows)] use std::os::windows::prelude::*;
-#[cfg(windows)] use std::fs::OpenOptions;
+
+#[cfg(any(target_os = "macos",
+          target_os = "ios",
+          target_os = "freebsd",
+          target_os = "dragonfly",
+          target_os = "openbsd",
+          target_os = "netbsd",
+          target_os = "bitrig",
+          target_os = "solaris",
+          target_os = "haiku"))]
+use libc::{c_char, c_int, timeval};
```

Comment:
Is this top-level import still needed?

Reply:
Probably not, but I *thought* I actually tested this on Mac... will double check what happened here..